### PR TITLE
refactor: fix lint and formatting issues

### PR DESCRIPTION
**Pull request by saikumarrs**
## Description of the change

> Description here

## Type of change
- [x] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

N/A

### Development

- [x] Lint rules pass locally
- [x] The code changed/added as part of this pull request has been covered with tests
- [x] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [x] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

**Comment by utsabc** [comment]
I see we have made changes to many destination in terms code how whats the testing plan for this change?

**Comment by shrouti1507** [comment]
> I see we have made changes to many destination in terms code how whats the testing plan for this change?

not sure, should we plan a bugbash!

File: sample.env [hunk 1]
```diff
@@ -7,5 +7,4 @@ CONFIG_BACKEND_URL="https://api.rudderstack.com"
 REDIS_HOST = localhost
 REDIS_PORT = 6379
 REDIS_PASSWORD = 123
-REDIS_USERNAME = abc
```
Comment (krishna2020) [inline sample.env:10]: this field is required

File: src/.nvmrc
Comment (krishna2020) [inline src/.nvmrc:1]: why is this file deleted ?

Comment (saikumarrs) [reply src/.nvmrc:1]: This file is already present in the outermost level.

File: sample.env (hunk 1 above)
Comment (saikumarrs) [reply sample.env:10]: @shrouti1507 did you delete this?

Comment (shrouti1507) [reply sample.env:10]: don't think so. Some merge results I guess. Putting it back.

File: src/util/dynamicConfigParser.ts [hunk 2]
```diff
@@ -14,7 +14,7 @@ export class DynamicConfigParser {
     //  " {{message.traits.key  ||   \"email\" }} "
     //  " {{ message.traits.key || 1233 }} "
     const defFormat =
-      /^\s*\{\{\s*(?<path>[a-zA-Z_]([a-zA-Z0-9_]*\.[a-zA-Z_][a-zA-Z0-9_]*)+)+\s*\|\|\s*(?<defaultVal>.*)\s*\}\}\s*$/;
+      /^\s*{{\s*(?<path>[A-Z_a-z](\w*\.[A-Z_a-z]\w*)+)+\s*\|\|\s*(?<defaultVal>.*)\s*}}\s*$/;
```
Comment (krishna2020) [inline src/util/dynamicConfigParser.ts:17]: what's this change to do with EsLint ?

File: src/v0/destinations/adobe_analytics/transform.js
Comment (krishna2020) [inline src/v0/destinations/adobe_analytics/transform.js:1]: @yashasvibajpai Have you done sanity testing of adobe analytics & quick review on the changes here ?

File: src/util/dynamicConfigParser.ts (hunk 2 above)
Comment (saikumarrs) [reply src/util/dynamicConfigParser.ts:17]: ESLint replaced the redundant parts of the regex with the simplified version.

File: test/__tests__/integration/service.integration.test.ts [hunk 3]
```diff
@@ -37,7 +37,10 @@ const getDataFromPath = (pathInput) => {
 
 describe('Basic route tests', () => {
   test('successful features response', async () => {
-    const expectedData = JSON.parse(fs.readFileSync('features.json', 'utf8'));
+    const expectedData = JSON.parse(
+      fs.readFileSync(path.resolve(__dirname, '../../../src/features.json'), 'utf8'),
+    );
+    console.log(expectedData);
```
[the remark this answers is not in the document]
Comment (shrouti1507) [reply test/__tests__/integration/service.integration.test.ts]: addressed

**Review by krishna2020 — APPROVED** [review]

**Review by shrouti1507 — APPROVED** [review]

**Review by utsabc — APPROVED** [review]
